module/vision/BallDetector/src/BallDetector.cpp

```diff
@@ -159,14 +160,15 @@ namespace module::vision {
 
                         // Attach the measurement to the object (distance from camera to ball)
                         b.measurements.push_back(Ball::Measurement());
-                        b.measurements.back().rBCc       = cartesianToSpherical(b.cone.axis * distance);
-                        b.measurements.back().covariance = config.ball_angular_cov;
+
+                        // Spherical Coordinates (1/distance, phi, theta)
+                        b.measurements.back().rBCc       = inverseDistanceCartesianToSpherical(b.cone.axis * distance);
```

Comment:
What does this do? And what's an inverse distance?

Reply:
Wait is an inverse distance just 1/distance?

Reply:
But I still don't understand it all together.

Reply:
Some smart person told us that the localisation filter tracks inverse distance better than it tracks distance.

Reply:
The reason to do inverse distance is that your error in your measurement is more linear. The further away the ball gets the less confident we are in our distance measurement. With a normal constant in a covariance matrix if we we would say the distance is x ± 0.1m or something like that.
But really the further away it gets the bigger our error, when we have distance = `1 / (1 / x ±0.1) it means the larger x is, the larger the ± is